facade/src/main/java/org/jboss/pnc/facade/impl/BrewPusherImpl.java

```diff
@@ -121,6 +119,10 @@ public BuildPushResult brewPush(String id, BuildPushRequest buildPushRequest) th
                                 .build())
                 .collect(Collectors.toList());
 
+        if (pushed.stream().anyMatch(r -> r.getMessage().contains("BLACKLISTED/DELETED"))) {
```

Comment:
use the `pushedResponse.get(0)` before this if and everything will be much simpler.